code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -207,7 +207,7 @@
 			var/damage = rand(1, 9)
 			if (prob(90))
 				if (HULK in M.mutations)//HULK SMASH
-					damage += 14
+					damage = 15

```

Comment:
Technically nerfs the damage, 19 would be the average after adding 14 to the randomized damage above.